### ISSUE TITLE
No Mac version?

### ISSUE THREAD

**Issue by eagsalazar** [LyricWulf/Embers#167]
**Is your feature request related to a problem? Please describe.**
Release docs say "mac build coming soon" but no build 1.5 years later...

**Describe the solution you'd like**
dmg

**Describe alternatives you've considered**
¯\_(ツ)_/¯

**Comment by CowboyBepBop** [comment]
The mac build is available in the discord server https://discord.gg/2HTve48

**Comment by Andrew-J-Larson** [comment]
Notice the latest release notes:

> ### Mac
> Mac build coming soon!

**Comment by CowboyBepBop** [comment]
Yes the mac build hasn't been released on the github but is available on our discord server https://discord.com/invite/2HTve48
You just have to grab the mac role and download it from the #mac-news channel

**Comment by Andrew-J-Larson** [comment]
> Yes the mac build hasn't been released on the github but is available on our discord server https://discord.com/invite/2HTve48 You just have to grab the mac role and download it from the #mac-news channel

Yes, but that's a preview/release candidate, right?

**Comment by CowboyBepBop** [comment]
It's marked as final release candidate but is the latest mac release available
